pkg/cap/cluster/anchor/host.go

```diff
@@ -0,0 +1,194 @@
+package anchor
+
+import (
+	"context"
+	"sync"
+
+	"capnproto.org/go/capnp/v3"
+	"capnproto.org/go/capnp/v3/rpc"
+	"capnproto.org/go/capnp/v3/server"
+	"github.com/libp2p/go-libp2p-core/peer"
+	api "github.com/wetware/ww/internal/api/cluster"
+	"github.com/wetware/ww/pkg/cap/cluster"
+	"github.com/wetware/ww/pkg/vat"
+)
+
+var (
+	Capability = vat.BasicCap{
+		"hostAnchor/packed",
+		"hostAnchor"}
+	defaultPolicy = server.Policy{
+		// HACK:  raise MaxConcurrentCalls to mitigate known deadlock condition.
+		//        https://github.com/capnproto/go-capnproto2/issues/189
+		MaxConcurrentCalls: 64,
+		AnswerQueueSize:    64,
+	}
+)
+
+type HostAnchor struct {
+	Peer peer.ID
+	Vat  vat.Network
+
+	Client api.Host
+
+	once sync.Once
```

Comment:
Nit:  I suggest placing this directly above `Client`, so as to indicate that it is protecting that field.  Also, `Client` is rather sensitive since it requires synchronization -- is it possible to not export it?